src/core/aio.c

```diff
@@ -505,6 +508,7 @@ nni_aio_list_active(nni_aio *aio)
 static void
 nni_aio_expire_add(nni_aio *aio)
 {
+	/*
```

Comment:
Please don't just comment out code -- remove it.   It makes it easier to review what you're changing, and keeps us from accruing cruft.